app/views/catalog/_results_pagination.html.erb

```diff
@@ -1,9 +1,9 @@
 <% if show_pagination? and @response.total_pages > 1 %>
  <div class="row record-padding">
   <div class="col-md-12">
-    <div class="pagination">
+    <nav class="pagination" role="region" aria-label="<%= t('views.pagination.aria.container_label') %>">
```

Comment:
I've seen a few web resources that specify using `role="navigation"` for major groups like this. Is there somewhere I can learn more about  the rationale for `role="region"`?

I've been looking at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role and http://www.a11ymatters.com/pattern/pagination/

Reply:
@mejackreed I believe the feedback from the consultants stated that the pagination is "secondary" navigation and not "main" navigation, so adding the role of region was to fill in for IE–me thinks-and the spotty support of `<nav>` with assistive tech with that browser. 

I'll go back and dig up the report, but I believe the idea is that if every grouping of links has `role=navigation` then nothing stands out as more important than the other. You know? What's the hierarchy of navigation on the page? menu bar > facets > pagination?

Reply:
Is this a relevant ref: https://www.w3.org/WAI/GL/wiki/Using_the_region_role_to_identify_a_region_of_the_page

Reply:
Sounds good. Thanks for the context and understanding @mtribone